docs/porting/configuration-differences.md

```diff
@@ -259,8 +315,11 @@ The following table contains the differences in libssl configuration options AWS
   <td rowspan=2>
   <p>
     <span>
-      <a href="https://www.openssl.org/docs/manmaster/man3/X509_VERIFY_PARAM_get0_peername.html">
+      <a href="https://github.com/aws/aws-lc/blob/e91524c10ad698fd56f77289ba3430baf3c7af64/include/openssl/ssl.h#L3089-L3096l">
         SSL_set_hostflags<br>
+      </a>
+      <br>
+      <a href="https://www.openssl.org/docs/manmaster/man3/X509_VERIFY_PARAM_get0_peername.html">
```

Comment:
Can we add the required documentation to our header files and link to that instead of OpenSSL?

Reply:
I opted for this only because the functions were all over the place in the headers, but did similar things. I can add documentation to these functions, but I'll have to create a follow up PR to update the links.

Reply:
I noticed upstream had some commits to finish documenting `x509.h`: https://github.com/google/boringssl/commit/5bed5b9aaab4edc8c0ee62493b6e760f9f7a3457
The documentation is much more robust here, should we prioritize taking these commits instead?

Reply:
Stole some documentation from https://github.com/google/boringssl/commit/5bed5b9aaab4edc8c0ee62493b6e760f9f7a3457 prematurely and added a "TODO" to update the links once we officially take it. There's a large assortment of commits in the header file so we'll want to take these incrementally.

Updating the related functions now is better than the current state of no documentation. Will create a follow up PR to link the table to the new text.